src/services/identity/SitesService.ts

```diff
@@ -125,64 +227,24 @@ class SitesService {
       },
     })
 
-    if (!site) {
-      return {
-        staging: "",
-        prod: "",
-      }
-    }
-
-    const deployment = site?.deployment
-    const output: SiteUrls = {
-      staging: deployment?.stagingUrl ? deployment.stagingUrl : "",
-      prod: deployment?.productionUrl ? deployment.productionUrl : "",
+    // Note: site may be null if the site does not exist
+    let siteUrls: SiteUrls = {
+      staging: site?.deployment?.stagingUrl ? site.deployment.stagingUrl : "",
+      prod: site?.deployment?.productionUrl
+        ? site.deployment.productionUrl
+        : "",
```

Comment:
we can use `??` or `||` - both easier to read

Reply:
Improved in 42e73df.